packages/components/select/src/select.vue

```diff
@@ -156,7 +156,6 @@
                 :class="[nsSelect.e('input'), nsSelect.is(selectSize)]"
                 :disabled="selectDisabled"
                 :autocomplete="autocomplete"
-                :style="inputStyle"
```

Comment:
It seems that `inputStyle` is only used here, and perhaps the related code can be removed at the same time.

In addition, since the Select component and the SelectV2 component use the same style, it may be necessary to handle the SelectV2 component at the same time.